LocationServices/LocationServicesUITests/SearchUITests.swift

```diff
@@ -25,15 +25,15 @@ final class SearchUITests: LocationServicesUITests {
         super.tearDown()
     }
     
-    func testSearchByAddressName() throws {
-        let app = startApp()
-        let _ = UITestExploreScreen(app: app)
-            .tapSearchTextField()
-            .waitForSearchRootView()
-            .type(text: Constants.addressName)
-            .tapKeyboardReturnButton()
-            .waitForResultsInTable()
-    }
+//    func testSearchByAddressName() throws {
+//        let app = startApp()
+//        let _ = UITestExploreScreen(app: app)
+//            .tapSearchTextField()
+//            .waitForSearchRootView()
+//            .type(text: Constants.addressName)
+//            .tapKeyboardReturnButton()
+//            .waitForResultsInTable()
+//    }
     
```

Comment:
If these tests aren't coming back, they should be removed. It's ok to leave them for now if they're just temporarily commented out though.

Reply:
removed